h3i/src/client/sync_client.rs

```diff
@@ -57,6 +58,15 @@ struct SyncClient {
     stream_parsers: StreamParserMap,
 }
 
+impl SyncClient {
+    fn new(expected_frames: Option<ExpectedFrames>) -> Self {
```

Comment:
Should this be part of the `Client` trait or no?

Reply:
Unfortunately not, the async client also implements `Client` but its `new` function returns an `AsyncClient`

Reply:
Well, we could probably make `Client<T>` but that will be messier without real benefit IMO

Reply:
No biggie